dbos/_sys_db.py

```diff
@@ -189,6 +189,9 @@ def __init__(self, config: ConfigFile):
             host=config["database"]["hostname"],
             port=config["database"]["port"],
             database="postgres",
+            query={
```

Comment:
What does this do?

Reply:
This fills the "`application_name`" column in the screenshot attached to the PR description. I found it very helfup.

Reply:
This needs a comment at the very least